src/components/ConfirmationPage/ConfirmationPage.js

```diff
@@ -0,0 +1,298 @@
+						<Flex align='center' wrap='wrap' my={2}>
+							<Text
+								color={textColorLight[mode]}
+								my={2}
+								w={["100%", "150px", "200px", "200px"]}
+							>
+								Stash Account
+							</Text>
+							<Select
+								placeholder='Select Account'
+								w={[
+									"calc(100%)",
+									"calc(100% - 150px)",
+									"calc(100% - 200px)",
+									"calc(100% - 200px)"
+								]}
+							>
+								{props.stashOptions.map((doc, index) => {
+									return (
+										<option
+											style={{ color: "#000", background: "#FFF" }}
+											value={doc.value}
+											key={index}
+										>
+											{doc.option}
+										</option>
+									);
+								})}
+							</Select>
+						</Flex>
+						<Flex align='center' wrap='wrap' my={2}>
+							<Text
+								color={textColorLight[mode]}
+								my={2}
+								w={["100%", "150px", "200px", "200px"]}
+							>
+								Controller Account
+							</Text>
+							<Select
+								placeholder='Select Account'
+								w={[
+									"calc(100%)",
+									"calc(100% - 150px)",
+									"calc(100% - 200px)",
+									"calc(100% - 200px)"
+								]}
+							>
+								{props.controllerOptions.map((doc, index) => {
+									return (
+										<option
+											value={doc.value}
+											key={index}
+											style={{ color: "#000", background: "#FFF" }}
+										>
+											{doc.option}
+										</option>
+									);
+								})}
+							</Select>
+						</Flex>
+						<Flex align='center' wrap='wrap' my={2}>
+							<Text
+								color={textColorLight[mode]}
+								my={2}
+								w={["100%", "150px", "200px", "200px"]}
+							>
+								Risk Preference
+							</Text>
+							<Badge
+								mx={1}
+								px={3}
+								py={1}
+								fontSize='sm'
+								variantColor={getRiskLevelColor(props.riskPreference)}
+							>
+								{getRiskLevel(props.riskPreference)}
+							</Badge>
+						</Flex>
+						<Box w='100%' p={2} mt={6} h='50vh' overflow='auto'>
+							{props.validatorsList.map((validator, index) => {
+								return (
+									<ValidatorTile
+										key={index}
+										name={validator.name}
+										amount={validator.amount}
+										currency={props.currency}
+										avatar={validator.avatar}
+										colorMode={props.colorMode}
+									/>
+								);
+							})}
+						</Box>
+						<Text
+							textAlign='center'
+							color={textColorLight[mode]}
+							my={4}
+							fontSize='xs'
+						>
+							* the estimated daily earnings are indicative. Actual rewards may
+							vary.
+						</Text>
+						<Box
+							p={4}
+							my={4}
+							w='100%'
+							bg={messageBoxColors[mode].bg}
+							borderColor={messageBoxColors[mode].border}
+							borderWidth='1px'
+							rounded='md'
+						>
+							<Text
+								fontSize='xs'
+								color={messageBoxColors[mode].text}
+								textAlign='center'
+							>
+								Fees of 10.000 milli KSM will be applied to the submission and
```

Comment:
The fees can vary - can we have a variable for this as well? - Just because I wouldn't want this to be missed - when it's hard-coded it's easy to miss while integrating